R2Streamer.podspec

```diff
@@ -1,20 +1,20 @@
 Pod::Spec.new do |s|
 
   s.name         = "R2Streamer"
-  s.version      = "1.2.5"
+  s.version      = "2.0.0-alpha.1"
   s.license      = "BSD 3-Clause License"
   s.summary      = "R2 Streamer"
   s.homepage     = "http://readium.github.io"
   s.author       = { "Aferdita Muriqi" => "aferdita.muriqi@gmail.com" }
-  s.source       = { :git => "https://github.com/readium/r2-streamer-swift.git", :tag => "1.2.5" }
+  s.source       = { :git => "https://github.com/readium/r2-streamer-swift.git", :tag => "2.0.0-alpha.1" }
```

Comment:
also here and below we should refer to our fork?

Reply:
we don't use cocoapods, do we really need to make our own pod for that?

Reply:
lol no... I just meant for completeness to replace the link to point to our fork instead of readium's, that's it :)